Actuator.m

```diff
@@ -198,8 +198,32 @@ function tune_pid_controller(obj, saturation, p_gain, i_gain, dv_gain, de_gain)
             configuration = [p_gain, i_gain, dv_gain, de_gain , saturationL_H];
             obj.write_multi_registers(133, 6, configuration);  % PC_PGAIN 133, num consecutive registers 6, register values  configuration
         end
+
         
+        %% Change Motor Direction
+        % Sets the motor direction to 0 or 1
+        function set_direction(obj, direction)
+            obj.write_register(152, direction);   %POS_SIGN
```

Comment:
This should be done through control register 1.
Reasoning being it also zeroes the motor at the same time, while making sure the POS_SIGN register is set to a valid value. Writing directly to the register can end up with invalid values.

Reply:
Should I change this to write to ctrl_reg_0 to 4 or 12 to trigger the zeroing at the same time?